generic_config_updater/field_operation_validators.py

```diff
@@ -27,7 +29,12 @@ def get_asic_name():
     if asic_type == 'cisco-8000':
         asic = "cisco-8000"
     elif asic_type == 'mellanox' or asic_type == 'vs' or asic_type == 'broadcom':
-        proc = subprocess.Popen(GET_HWSKU_CMD, shell=True, universal_newlines=True, stdout=subprocess.PIPE)
+        get_hwsku_cmds = []
+        if scope == HOST_NAMESPACE:
+            get_hwsku_cmds = ["sonic-cfggen", "-d", "-v", "DEVICE_METADATA.localhost.hwsku"]
+        else:
```

Comment:
This may not be really required as we have the hwsku in the hostname space and asic namespace. But it should be ok as we have the same in all namespaces.